doc/user/content/get-started/_index.md

```diff
@@ -60,13 +60,21 @@ this exact purpose.
 
 ### Standard SQL support
 
-Materialize follows the SQL standard (SQL-92) implementation, so you interact
-with it like any relational database: using SQL. You can build complex
-analytical workloads using **[any type of join](/sql/select/join/)** (including
-non-windowed joins and joins on arbitrary conditions), but you can also
-leverage exciting new SQL patterns enabled by streaming like
-[**Change Data Capture (CDC)**](/integrations/#databases),
-[**temporal filters**](/sql/patterns/temporal-filters/), and [**subscriptions**](/sql/subscribe/).
+You can interact with Materialize using SQL. You can build complex analytical
+workloads using **[various types of join](/sql/select/join/)** (including
+non-windowed joins and joins on arbitrary conditions) as well as leverage new
+SQL patterns enabled by streaming like [**Change Data Capture
+(CDC)**](/integrations/#databases), [**temporal
+filters**](/sql/patterns/temporal-filters/), and
+[**subscriptions**](/sql/subscribe/).
+
+Materialize follows the SQL standard (SQL-92) implementation and strives for
+compatibility with the PostgreSQL dialect rather than aiming for compatibility
+with a specific version of PostgreSQL. This allows Materialize to add the latest
+PostgreSQL features even as some features of specific versions of PostgreSQL are
+missing. Performance for supported features may differ between Materialize and
+PostgreSQL, such as with [window
+functions](/transform-data/idiomatic-materialize-sql/appendix/window-function-to-materialize).
```

Comment:
```suggestion
Materialize follows the SQL standard (SQL-92) implementation and aims for
compatibility with the PostgreSQL dialect. It **does not** aim for
compatibility with a specific version of PostgreSQL. This means that
Materialize might support syntax from any released PostgreSQL version, but does
not provide full coverage of the PostgreSQL dialect. The implementation and
performance of specific features (like [window functions](/transform-data/idiomatic-materialize-sql/appendix/window-function-to-materialize))
might also differ, because Materialize uses an entirely different database engine
based on [Timely and Differential Dataflow](/get-started/#incremental-updates).
```

Reply:
Love the suggestion ❤️❤️ ❤️    Let me poke the people whose itch we're trying to scratch to check that they're good with this.